ckanext/charts/templates/scheming/form_snippets/chart_more_info_button.html

```diff
@@ -0,0 +1,77 @@
+<div class="form-group control-full control-large">
+    <div class="controls">
+        <button class="btn btn-primary" type="button" data-bs-toggle="modal" data-bs-target="#moreInfoModal">
+            More info
+        </button>
+    </div>
+</div>
+
+
+<div class="modal fade" id="moreInfoModal" tabindex="-1" role="dialog" aria-labelledby="moreInfoModalLabel" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="moreInfoModalLabel">Supported Date Formats</h5>
+                <button type="button" class="btn-close close" data-bs-dismiss="modal" aria-label="Close">
+                </button>
+            </div>
+            <div class="modal-body">
+                <div class="date-formats-container">
+                    <div class="date-formats-container__row">
+                        <div class="date-formats-container__column">
+                            <p class="date-formats-text"><strong>Timestamp Format</strong></p>
+                            <ul class="date-formats-list">
+                                <li class="date-formats-item">YYYY-MM-DD</li>
```

Comment:
my suggestion is to convert this into a dict and traverse key, values
```py
{% set datetime_mapping = {
  "YYYY-MM-DD": "2023-07-24",
  "YYYY/MM/DD" : "2023/07/24"
} %}
```
and thence create a table and add tr with 2 td in each iteration, like
```
<table class="table">
  <thead>
      <tr>
          <th>{{ _("Format") }}</th>
          <th>{{ _("Example") }}</th>
      </tr>
  </thead>

  {% for format, example in datetime_mapping.items() %}
      <tr>
          <td>{{ format }}</td>
          <td>{{ example }}</td>
      </tr>
  {% endfor %}
</table>
```

because now it would be hard to edit, cause you need to correlate format with an example.